Added Flags for port and filter list form external file (#10)
jacobbednarz/go-csp-collector: README.md

### Running

```sh
$ CGO_ENABLED=0 go build csp_collector.go
$ ./csp_collector
```

> The `CGO_ENABLED=0` flag makes the build compatible with alpine linux in a docker container

Reviewer: ```suggestion
The `CGO_ENABLED=0` flag makes the build compatible with alpine linux in a docker container.
```

> ### Command Line Options

Reviewer: ```suggestion
### Command Line Options

```

$ ./csp_collector
```

#### Building for Docker

You will either need to build within a docker container for the purpose, or use  `CGO_ENABLED=0` flag
to make the build compatible with alpine linux in a docker container.

```sh
$ CGO_ENABLED=0 go build csp_collector.go
```

### Command Line Options

| Flag  | Description |
|-------|:------------|
|version|Shows the version string before exiting|
|debug  |Runs in debug mode producing more verbose output|
|port	|Port to run on, default 8080|

> |filter-file|Reads the filter list from the specified file. Note on filter per line|

Reviewer: ```suggestion
|filter-file|Reads the blocked URI filter list from the specified file. Note one filter per line|
```
